### DISCUSSION TITLE
Err: The ML-framework you used is paddlepaddle, not tensorflow!

### DISCUSSION THREAD

**Discussion by btrotta** [PaddlePaddle/PaddleSpatial#169] [BaiduKDDCup2022] [answered]
I am using tensorflow and have set `"framework": "tensorflow"` in `prepare.py`. However when I submit, I get the following error:
```
Err: The ML-framework you used is paddlepaddle, not tensorflow! 
```

**Comment by seeingisbelieveing** [comment]
Dear participant, could you please provide your team name and submission date? We will check it, Thx.

**Reply by btrotta** [reply]
Thanks. Team name: BT. Submission date: 2022-06-13 09:53

**Reply by ramber1836** [reply]
I will fix this issue and let you know if it's ready. Thanks.

**Reply by ramber1836** [reply]
The online test has been updated, you can resubmit your code to check if the problem is solved. 
Thanks.

**Reply by btrotta** [reply]
@ramber1836 I am still getting the same error with my latest submission at 2022-06-15 18:38

**Reply by AlvinAi96** [reply]
> The online test has been updated, you can resubmit your code to check if the problem is solved. Thanks.

@ramber1836 I just come across the same problem as @btrotta. Would you like to double-check whether you updated the online test code?

**Reply by ramber1836** [reply]
My mistakes, I will update the online test soon. Let you know when it is ready.

**Comment by ramber1836** [comment]
@btrotta The online test has been updated.

**Reply by AlvinAi96** [reply]
> @btrotta The online test has been updated.
Error was reported again. my team name: Dynamo.
![image](https://user-images.githubusercontent.com/42187008/174004166-0e42cd42-a19a-4b62-81be-2f35eeb2e785.png)

**Reply by ramber1836** [reply]
Remove the unnecessary scripts within your submitted zip file and try again.

**Reply by AlvinAi96** [reply]
Is caused by importing my lgb and xgb library? Could you point out in details.
If not convenient, please inform me by email alvinai9603@outlook.com. 
Thanks!

**Reply by btrotta** [reply]
@ramber1836 Thanks for your help, I am no longer getting this error.

**Reply by ramber1836** [reply]
> Is caused by importing my lgb and xgb library? 

Yes, I think so.

**Reply by fersebas** [reply]
Could you please also updated the github code? thanks

**Comment by ramber1836** [comment, marked as answer]
The ML-framework usage is determined by the backend code inspector, which is not public. 
To avoid the ML-framework mis-usage error, you should make sure that all python scripts are necessary for your model and the real utilized ML-framework is consistent with the ML-framework you claimed.
